### ISSUE TITLE
Auto superscript or subscript 

### ISSUE THREAD

**Issue by amourchaleur** [rugk/unicodify#66]
Suggest for auto superscript or subscript for text such as 10th or 1st (in thunderbird).

**Comment by rugk** [comment]
Please mind to fill out the issue template you choose, when you open an issue the next time. This makes it easier to troubleshoot issues or understand feature requests etc.

As you seem to come from #30, I interpret this issue as including these subscript such as 10th as an auto-correct feature.

**Comment by tdulcet** [comment]
> Suggest for auto superscript or subscript for text such as 10th or 1st

OK, thanks for the feature request. I am assuming you want [ordinal numerals](https://en.wikipedia.org/wiki/Ordinal_numeral) like 10th and 1st to autocorrect to 10ᵗʰ and 1ˢᵗ respectfully for the auto superscript. What did you want for the auto subscript?

Users can already manually do superscript from the context menu. (It does not currently work in Thunderbird because of [Bug 1716976](https://bugzilla.mozilla.org/show_bug.cgi?id=1716976).) However, I am not aware of a subscript Unicode font, so auto subscript would not currently be possible.

> (in thunderbird).

The Firefox, Chrome and Thunderbird versions of this add-on all share the same code.

**Comment by amourchaleur** [comment]
Yup, auto superscript like 1st and 2nd. Correction: don't normally use subscript so don't have examples. Superscript is commonly used for typing dates etc. 

Thank you.

**Comment by amourchaleur** [comment]
Hi, would the auto superscript be implemented anytime soon? :) Thanks!

**Comment by tdulcet** [comment]
Sorry, I do not have any plans to implement this any time soon. Our priority is currently on improving the autocorrect feature so it can be enabled by default (see #4 and #64).

However, pull requests are always welcome! It should be relatively easy to implement, which is why I added the "good first issue" label. There are general instructions [here](https://github.com/rugk/unicodify/blob/main/CONTRIBUTING.md#coding-guidelines). The [`Intl.PluralRules`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/PluralRules) API might also be helpful.

**Comment by amourchaleur** [comment]
Oh I see, it's okay, not good with coding. Thanks for the reply!

**Comment by tdulcet** [comment]
I am not sure if this helps with your use case, but the extension now supports doing subscript from the context menu: 
![image](https://github.com/rugk/unicodify/assets/15305150/3f9da2ce-482d-410b-9319-9653d58029f9)
This will be included with the next version of the add-on.
